package.json

```diff
@@ -15,7 +15,9 @@
     "prod-build": "npm run build-themes && ng build --aot --prod",
     "postinstall": "webdriver-manager update && bash ./tools/fetch-assets.sh",
     "publish-prod": "npm run build-themes && ng build --aot --prod && firebase use material-angular-io && firebase deploy",
-    "publish-dev": "npm run build-themes && ng build --aot --prod && firebase use material2-docs-dev && firebase deploy"
+    "publish-dev": "npm run build-themes && ng build --aot --prod && firebase use material2-docs-dev && firebase deploy",
+    "publish-v5-prod": "npm run build-themes && ng build --aot --prod && firebase use v5-material-angular-io && firebase deploy",
+    "publish-v5-dev": "npm run build-themes && ng build --aot --prod && firebase use material2-docs-5 && firebase deploy"
```

Comment:
File an issue saying we should create a deploy script for this rather than adding more npm commands over time?

Reply:
Added file `tools/deploy.sh` and updated `package.json`